lisk-tree cannot resolve cryptography package

### Expected behavior
Install lisk-tree package should be able to resolve cryptography

### Actual behavior
```
FAIL src/modules/pos/validator/components/ValidatorProfile/ValidatorProfile.test.js

  ● Test suite failed to run
    Cannot find module '../../../lisk-cryptography' from 'node_modules/@liskhq/lisk-tree/dist-node/merkle_tree/calculate.js'
    Require stack:
      node_modules/@liskhq/lisk-tree/dist-node/merkle_tree/calculate.js
      node_modules/@liskhq/lisk-tree/dist-node/index.js
      node_modules/@liskhq/lisk-client/dist-node/index.js
      src/modules/wallet/utils/account.js
      src/modules/wallet/components/walletVisualWithAddress/index.js
      src/modules/pos/validator/components/ValidatorProfile/ValidatorProfile.js
      src/modules/pos/validator/components/ValidatorProfile/ValidatorProfile.test.js
      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (node_modules/@liskhq/lisk-tree/src/merkle_tree/calculate.ts:15:1)
```

### Which version(s) does this affect? (Environment, OS, etc...)
6.0.0-alpha.9-

--- a/elements/lisk-tree/src/merkle_tree/calculate.ts
+++ b/elements/lisk-tree/src/merkle_tree/calculate.ts
@@ -12,7 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { utils } from '../../../lisk-cryptography';
+import { utils } from '@liskhq/lisk-cryptography';
 import { LEAF_PREFIX } from './constants';
 import { calculatePathNodes, ROOT_INDEX } from './utils';
 

--- a/elements/lisk-tree/src/merkle_tree/merkle_tree.ts
+++ b/elements/lisk-tree/src/merkle_tree/merkle_tree.ts
@@ -14,7 +14,7 @@
 /* eslint-disable no-bitwise */
 /* eslint-disable @typescript-eslint/prefer-for-of */
 
-import { utils } from '../../../lisk-cryptography';
+import { utils } from '@liskhq/lisk-cryptography';
 import {
 	LAYER_INDEX_SIZE,
 	NODE_HASH_SIZE,

